invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities-carousel/CommunitiesCarousel.js

```diff
@@ -0,0 +1,174 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2016-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import _truncate from "lodash/truncate";
+import { i18next } from "@translations/invenio_communities/i18next";
+import { http } from "../../api/config";
+import { withCancel } from "react-invenio-forms";
+import { Transition, Container, Grid, Header, Item, Icon } from "semantic-ui-react";
+import CarouselItem from "./CarouselItem";
+
+export default class CommunitiesCarousel extends Component {
```

Comment:
not a blocker: did we consider to use any existing carousel library?

Reply:
I was thinking about it, but figured it didn't make sense, when SUI already has what we need with just a few tweaks